tests/api_v2/test_util_methods.py

```diff
@@ -218,21 +213,30 @@ def test_filter_by_project(self):
         """
         sw = dict(
             code="Tester",
-            engine="tk_engine_tester",
+            engine="tk-maya",
```

Comment:
I found tk_engine_tester<number> to be confusing, so I changed those to nice graspable names